### PR TITLE
Add blog post

### PR DESCRIPTION
**Pull request by stevekinney**
## What does this PR do?

It adds a blog post about how our users can use the beta of the new Temporal UI today!

### CODE REVIEW

File: blog/2022-03-24-web-v2-beta-announcement.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+author_image_url: https://avatars.githubusercontent.com/u/251000?v=4
+---
+
+# Temporal UI Beta is Now Available
+
+We're excited to announce the public beta of the new Temporal UI, which is available today as an opt-in feature for our open source users. We’d love [your feedback](https://github.com/temporalio/ui/issues).
+
+![Screenshot of Web v2](https://user-images.githubusercontent.com/251000/159999018-d82dfe25-394b-4332-b6e8-a4fedeceec34.pn)
+
+The new Temporal UI is a ground-up re-write. This first release aims for parity with the current UI. Our goal was to lay the foundation for an architecture that we can iterate on. We'll continue to ship features and functionality of the foreseeable future. We also think it looks pretty nice.
```
Comment (djmagee) [inline blog/2022-03-24-web-v2-beta-announcement.md:19]:
```suggestion
The new Temporal UI is a ground-up rewrite. This first release aims for parity with the current UI. Our goal was to lay the foundation for an architecture that we can iterate on. We'll continue to ship features and functionality for the foreseeable future. We also think it looks pretty nice.
```

File: blog/2022-03-24-web-v2-beta-announcement.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+We addressed dozen of quality of life issues and improved the performance of the user interface. We built the new web experience with [Svelte](https://svelte.dev/), which supports many of the best practices of modern web development. Svelte does the majority of its magic at compile time which limits the amount of code shipped to operate the application at run time. The result is a significantly smaller application bundle and noticeably snappier performance.
```
Comment (djmagee) [inline blog/2022-03-24-web-v2-beta-announcement.md:21]:
```suggestion
We addressed dozens of quality-of-life issues and improved the performance of the user interface. We built the new web experience with [Svelte](https://svelte.dev/), which supports many of the best practices of modern web development. Svelte does the majority of its magic at compile time, which limits the amount of code shipped to operate the application at run time. The result is a significantly smaller application bundle and noticeably snappier performance.
```

File: blog/2022-03-24-web-v2-beta-announcement.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+
+![Event Summary View](https://user-images.githubusercontent.com/251000/160000073-fcc79ef6-4be3-4f4a-98c8-2831007a26f6.png)
+
+We snuck in some new features along the way. In addition to a fresh look and feel, you'll notice that it's easier to see if you're using [data converter](https://docs.temporal.io/docs/concepts/what-is-a-data-converter/). We store filter settings and queries in the URL. This means that sharing what you're currently looking at with your team is as easy as sharing a link.
```
Comment (djmagee) [inline blog/2022-03-24-web-v2-beta-announcement.md:25]:
```suggestion
We snuck in some new features along the way. In addition to a fresh look and feel, you'll notice that it's easier to see whether you're using a [Data Converter](https://docs.temporal.io/docs/concepts/what-is-a-data-converter/). We store filter settings and queries in the URL. This means that sharing what you're currently looking at with your team is as easy as sharing a link.
```

File: blog/2022-03-24-web-v2-beta-announcement.md [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+
+The new user interface will eventually be the default experience for working with Temporal in the browser. In the mean time, we'd love for you to try it out as we continue to iterate on it. Taking the new UI for a spin is incredibly easy. If you pull down the latest version of Temporal’s [docker-compose](https://github.com/temporalio/docker-compose), it’s already running alongside the current UI but on Port 8080.
```
Comment (djmagee) [inline blog/2022-03-24-web-v2-beta-announcement.md:27]:
```suggestion
The new user interface will eventually be the default experience for working with Temporal in the browser. In the meantime, we'd love for you to try it out as we continue to iterate on it. Taking the new UI for a spin is incredibly easy. If you pull down the latest version of Temporal’s [docker-compose](https://github.com/temporalio/docker-compose), it’s already running alongside the current UI but on port 8080.
```

File: blog/2022-03-24-web-v2-beta-announcement.md [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+
+![Event Full View](https://user-images.githubusercontent.com/251000/160000081-77f1b8b8-2c60-44b4-ab41-7461bcf9b558.png)
+
+Big changes to the user interface can be tricky when you’re running mission-critical software like Temporal in production. For the time being, you can run both experiences in parallel. We’ll continue to support the existing UI as we polish and iterate on the new UI. But, over the next few weeks, we’ll begin to direct new users to use the new UI. We’ll be releasing new versions multiple times a week as we get your feedback and solve for your use cases. On September 30, 2022, we’ll remove the current UI.
```
Comment (djmagee) [inline blog/2022-03-24-web-v2-beta-announcement.md:31]:
```suggestion
Big changes to the user interface can be tricky when you’re running mission-critical software like Temporal in production. For the time being, you can run both experiences in parallel. We’ll continue to support the existing UI as we polish and iterate on the new UI. But, over the next few weeks, we’ll begin to direct new users to use the new UI. We’ll release new versions multiple times each week as we get your feedback and solve for your use cases. On September 30, 2022, we’ll remove the current UI.
```

**Review by rylandg — APPROVED** [review]
LGTM

File: blog/2022-03-24-web-v2-beta-announcement.md [hunk 6]
```diff
@@ -0,0 +1,33 @@
...
+
+![Event Full View](https://user-images.githubusercontent.com/251000/160000081-77f1b8b8-2c60-44b4-ab41-7461bcf9b558.png)
+
+Big changes to the user interface can be tricky when you’re running mission-critical software like Temporal in production. For the time being, you can run both experiences in parallel. We’ll continue to support the existing UI as we polish and iterate on the new UI. But, over the next few weeks, we’ll begin to direct new users to use the new UI. We’ll release new versions multiple times each week as we get your feedback and solve for your use cases. On September 30, 2022, we’ll remove the current UI.
+
+This is only the beginning for us. We have a lot of exciting features planned, now that we have this new architecture in place. We'd love to hear from you. If you encounter any issues or have any ideas for features that you'd like to see, file an issue in [our repository](https://github.com/temporalio/ui).
```
Comment (lorensr) [inline blog/2022-03-24-web-v2-beta-announcement.md:33]:
```suggestion
This is only the beginning for us. We have a lot of exciting features planned, now that we have this new architecture in place. We'd love to hear from you. If you encounter any issues or have any ideas for features that you'd like to see, please file an issue in [our repository](https://github.com/temporalio/ui)!
```

Minor tone idea ☺️

**Comment by lorensr** [comment]
Filename out of date

File: blog/2022-03-24-web-v2-beta-announcement.md (hunk 6 above)
Comment (djmagee) [reply blog/2022-03-24-web-v2-beta-announcement.md:33]: Per "please" in https://developers.google.com/style/word-list#letter-p ("Use _please_ only when you're asking for permission or forgiveness"), I recommend stetting.

Having said that, the guidelines for blog posts are somewhat looser than for documentation.

**Review by flossypurse — APPROVED** [review]

**Review by rylandg — APPROVED** [review]

**Pull request merged**